docs/docs/themes/api-reference.md

```diff
@@ -118,7 +118,7 @@ module.exports = {
     {
       resolve: "gatsby-plugin-compile-es6-packages",
       options: {
-        modules: ["gatsby-theme-developer"],
+        modules: ["NAME_OF_YOUR_THEME"],
```

Comment:
How about we solve this in a different way?

```suggestion
        // replace with the name of your theme
        // highlight-next-line
        modules: ["gatsby-theme-developer"],
```

Reply:
Yeah that also works^^